modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/build.gradle

```diff
@@ -39,4 +40,8 @@ wrapSoyTemplates {
 			soyJsFile.text = soyJsWrapperHeader + soyJsFile.text.replace("(typeof ddm == 'undefined') { var ddm = {}; }", "(typeof ddm == 'undefined') { window.ddm = {}; }") + soyJsWrapperFooter
 		}
 	}
+}
+
+testJS {
```

Comment:
@diogo-lins Tests failed on the CI. I think we are missing this line: https://github.com/brunobasto/liferay-portal/blob/master/modules/apps/foundation/frontend-js/frontend-js-web/build.gradle#L4